DESCRIPTION

```diff
@@ -1,12 +1,15 @@
 Type: Package
 Package: sasr
-Title: sasr
-Version: 0.0.0.9004
-Date: 2023-03-28
-Authors@R:
-    person("insightsengineering", , , "insightsengineering@example.com", role = c("aut", "cre"))
-Description: sasr.
-License: Apache License 2.0 | file LICENSE
+Title: SAS Interface in R
+Version: 0.1.0
+Date: 2023-04-26
+Authors@R: c(
+    person("Liming", "Li", , "liming.li@roche.com", role = c("aut", "cre")),
+    person("Daniel", "Sabanes Bove", , "daniel.sabanes_bove@roche.com", role = c("aut")),
+    person("F. Hoffmann-La Roche AG", role = c("cph", "fnd"))
+  )
+Description: SAS interface to execute SAS code remotely and retrieve results in R.
```

Comment:
sorry another comment, I think one sentence won't be sufficient for the description. suggest 3-4 sentences at least, also consider adding a link to the saspy interface that we are wrapping as a kind of reference